packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableInserter.ts

```diff
@@ -35,6 +36,10 @@ export default function createTableInserter(
             document
         ) as HTMLDivElement;
 
+        if (!zIndex !== undefined) {
```

Comment:
I think we can remove the `!` before zIndex.

Reply:
oops sorry 😓 
commit amended & rebased